src/Physics/Explodinator.cpp

```diff
@@ -0,0 +1,405 @@
+
+#include "Globals.h"
+#include "BlockInfo.h"
+#include "Blocks/BlockHandler.h"
+#include "Blocks/ChunkInterface.h"
+#include "Chunk.h"
+#include "ClientHandle.h"
+#include "Entities/FallingBlock.h"
+#include "LineBlockTracer.h"
+#include "Simulator/SandSimulator.h"
+
+
+
+
+
+namespace Explodinator
+{
+	const auto StepUnit = 0.3f;
+	const auto KnockbackFactor = 25U;
+	const auto StepAttenuation = 0.225f;
+	const auto TraceCubeSideLength = 16U;
+	const auto BoundingBoxStepUnit = 0.5f;
+
+	/** Returns how much of a Explosion Lazor's (tm) intensity the given block attenuates. */
+	static float GetAbsorption(const BLOCKTYPE Block)
```

Comment:
Style (everywhere in this file, apparently): Functino arguments should have the `a_` prefix.

Reply:
It looks like this though so an extra tag is superfluous I think, I don't suppose you'd consider a variation?

![Argument highlighting](https://user-images.githubusercontent.com/2939130/92311606-5eae8680-efb0-11ea-89cb-6a4a7ec4872d.png)

Reply:
Maybe if you consider changing to javaStyleIdentifierNames :P
Not everyone has such a smart IDE; GitHub definitely doesn't when reviewing the PRs.